README.md

```diff
@@ -89,13 +91,13 @@ someguy123:
 
 "I am the CEO of [@privex](https://steemit.com/@privex) - A VPS provider that accepts STEEM, is affordable, and [sells servers with a Zero Block Miss SLA](https://steemit.com/introduceyourself/@privex/we-are-privex-inc-we-accept-steem-protecting-your-privacy-in-the-cloud).
 
-I use @privex for my own witness. As a proof of their quality, I haven't missed a block since moving to them TBD, and I get almost 60 blocks per day. (As of May 2017)
+I use @privex as my own witness. As a proof of their quality, I haven't missed a block since moving to them TBD, and I get almost 60 blocks per day. (As of May 2017)
 
 You'll find plenty of people here on STEEM unaffiliated with us, praising our servers and network."
 
-You're free to use any other server provider. I don't recommend running a witness server on your home internet; it is unlikely to be reliable enough for that purpose.
+You're free to use any other service provider. I don't recommend running a witness server on your home internet; it is unlikely to be reliable enough for that purpose.
```

Comment:
I believe that server provider was already correct.

Service provider is much more generic.

Reply:
Agreed.